x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/map_filters_to_kql.test.ts

```diff
@@ -52,12 +52,12 @@ describe('mapFiltersToKql', () => {
     ]);
   });
 
-  test('should handle typesFilter, actionTypesFilter and alertStatusesFilter', () => {
+  test('should handle typesFilter, actionTypesFilter and ruleStatusesFilter', () => {
     expect(
       mapFiltersToKql({
         typesFilter: ['type', 'filter'],
         actionTypesFilter: ['action', 'types', 'filter'],
-        alertStatusesFilter: ['alert', 'statuses', 'filter'],
+        ruleStatusesFilter: ['alert', 'statuses', 'filter'],
```

Comment:
@XavierM 
```suggestion
        ruleStatusesFilter: ['rule', 'statuses', 'filter'],
```

Reply:
it should still be `alert` because that's still the name of our saved object type

Reply:
Ok great, it this part refers to the saved object `alert.attributes.alertTypeId`, then yep it should stay `alert`